scripts/pylint_extensions.py

```diff
@@ -794,6 +795,40 @@ def visit_functiondef(self, node):
             self.add_message('function-args-order-cls', node=node)
 
 
+class SingleCharAndNewlineAtEOFChecker(checkers.BaseChecker):
+    """Checker for single character files and newline at EOF."""
+
+    __implements__ = interfaces.IRawChecker
+    name = 'newline-at-eof'
+    priority = -1
+    msgs = {
+        'C0007': (
+            'Files should end in a single newline character.',
+            'newline-at-eof',
+            'Please enter a newline at the end of the file'),
+        'C0008': (
+            'Only one character in file',
+            'only-one-character',
+            'Files with only one character are not allowed'),
+    }
+
+    def process_module(self, node):
+        """Process a module.
+
+        Args:
+            node: astroid.scoped_nodes.Function. Node to access module content.
+        """
+
+        file_content = node.stream().readlines()
+        file_length = len(file_content)
+        print '@@@@@@@'
```

Comment:
What's this?

Reply:
Just something left over from my debugging :P Sorry I missed it.
Done!